apps/muddle-node/main.cpp

```diff
@@ -23,8 +23,8 @@
 #include "core/macros.hpp"
 #include "core/synchronisation/protected.hpp"
 #include "crypto/ecdsa.hpp"
+#include "http/http_server.hpp"
```

Comment:
renamed to http_server since I prefer to have file names match class names. Also moved functions into .cpp so not header only file.